versioned_docs/version-22.04/developer/developer-gorgone-client-server-communication.md

```diff
@@ -0,0 +1,93 @@
+---
+id: developer-gorgone-client-server-communication
+title : Managing client/server communication
+---
+
+Centreon Gorgone is used on the Central server, the Remote Servers, and the collectors.
+Gorgone is the process for a distributed architecture: it deploys the configuration generated from the central server to the collectors and 
+distributes external commands to the collectors. 
+
+Centreon recommends to use ZMQ protocol for the communication between two gorgoned processes. 
+When using ZMQ protocol, all communications are encrypted using symmetric-key encryption based on public/private keys from both client and 
+server. So you need to generate public/private keys to set the configuration.
+
+In a Centreon context, the client is the Gorgone daemon running on the Centreon Central and the servers are the daemon running on pollers.
```

Comment:
This is true, but some additional details could be nice
* central has a gorgone running (by default) as a client and can connect to gorgone servers running on remote servers and pollers
* remote servers have a gorgone running (by default) as a client and can connect to gorgone servers running on pollers